infrastructure/lookerstudio/lookerstudio-deployment-walkthrough.md

```diff
@@ -0,0 +1,89 @@
+## Marketing Data Engine Looker Studio Deployment Walkthrough
+
+These steps will generate the URL to deploy the Looker Studio Marketing Data Engine dashboard.
+
+## Environment Set-Up
+
+The installation script's library requirements are minimal however if you wish, you can configure a virtual environment.
+
+```sh
+cd infrastructure/lookerstudio
+python -m venv mde
+source ./mde/bin/activate
+
+pip install -r requirements.txt
+```
+
+This step includes the following:
+- Install Python virtual environment named `mde`
+- Activate the `mde` Python virtual environment
+- Install dependencies
+
+## Set-up Configuration
+
+The file `config.ini` needs to be updated for the project and datasets where your Marketing Data Engine Dataform is deployed.
+
+Specifically the following properties need to be set under the `[COMMON]` section:
+
+```
+[COMMON]
+# TODO: Replace the values in this section with your own
+
+project = project_id
+ga4_dataset = marketing_ga4_v1_prod
+ads_dataset = marketing_ads_v1_prod
+```
+
+You can use your favorite text editor such as vim, nano or even the built in editor in Cloud Console.
+
+![Editor](https://github.com/GoogleCloudPlatform/marketing-data-engine/raw/main/infrastructure/lookerstudio/images/editor.png)
```

Comment:
try just /infrastructure/lookersudio/images/editor.png. It should be more portable and I believe GitHub will put the right link there.

Reply:
I had originally used relative paths however the Cloud Shell tutorial interface requires fully qualified paths. It sounds like Carlos prefers we don't use Cloud Shell tutorials so if that is the case, I will revert back to relative paths.